File: Lab-Programs/LAB3/K-Largest/K-Largest.cpp
#include<iostream>
using namespace std;

int main()
{
 int n;
 cout<<"Enter the value of n:"<<endl;
 cin>>n;
 
 int a[n];
 cout<<"Enter the array elements :"<<endl;

 for(int i=0;i<n;i++)
 {
  cin>>a[i];
 }

 for(int i=0;i<n-1;i++)
 {
  for(int j=i+1;j<n;j++)
  {
    if(a[i]<a[j])
    {
     int t=a[i];
         a[i]=a[j];
         a[j]=t;
    }
  }
 }
 
 int k;
 cout<<"Enter the value of k:"<<endl;
 cin>>k;
 for(int i=0;i<k;i++)
 {
  cout<<a[i]<<" ";
 }
 cout<<endl;
return 0;
}

File: Lab-Programs/LAB3/K-Largest/k-largest.py
def k_largest_elements(arr, n, k):
    for i in range(k):
        for j in range(0, n-i-1):
            if arr[j+1] < arr[j]:
                arr[j], arr[j+1] = arr[j+1], arr[j]
    return arr[n-k:]

n = int(input("Enter number of elements you want"))
arr = []

for i in range(n):
    x = int(input("Enter number"))
    arr.append(x)

k = int(input("Enter value of k"))

print(k_largest_elements(arr, len(arr), k))
